aten/src/ATen/templates/TensorBody.h

```diff
@@ -363,6 +371,15 @@ class TORCH_API Tensor {
     return !at::impl::variable_excluded_from_dispatch();
   }
 
+  inline bool is_conj() const {
+    return impl_->is_conj();
+  }
+
+  // TODO: do I need to add this method directly on the Tensor to implement conj_view() in UnaryOps?
```

Comment:
Should this be private with `_set_conj()`? Users should never call this function directly right?